external/avro/src/main/scala/org/apache/spark/sql/avro/SchemaConverters.scala

```diff
@@ -37,16 +38,25 @@ object SchemaConverters {
 
   private lazy val nullSchema = Schema.create(Schema.Type.NULL)
 
+  /**
+   * Internal wrapper for SQL data type and nullability.
+   *
+   * @since 2.4
+   */
+  @DeveloperApi
   case class SchemaType(dataType: DataType, nullable: Boolean)
 
   /**
-   * This function takes an avro schema and returns a sql schema.
+   * Converts an Avro schema to a corresponding Spark SQL schema.
+   *
+   * @since 2.4
    */
+  @DeveloperApi
```

Comment:
I think we can just remove this on each method since we already noted in the outer object (and other methods became properly private)

Reply:
How about putting the `@DeveloperApi` in `object SchemaConverters`.

Reply:
I think so. We just need to mark the object.

Reply:
I am not sure what is the best practice for the annotation:
1. outer object/class only
2. methods only
3. both

I can see 3 cases in the current code..

Reply:
I think it's fine to mark the object alone.